### PR TITLE
Takeaway Weekend Challenge

### PR DESCRIPTION
**Pull request by katie210**

### CODE REVIEW

File: lib/sms.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+class SMS
+
+  def send_text(message)
```
Comment (hedudelgado) [inline lib/sms.rb]: The Readme is missing, remember to fill it up when you have a chance ^__^

Comment (hedudelgado) [inline lib/sms.rb]: Be careful, the sensitive data might be exposed :) (private)
Some methods may be moved to private as there is no need to call them to make the program work.

Comment (hedudelgado) [inline lib/sms.rb]: Remember to remove comments.

**Comment by mechacoach** [comment]
You had your takeaway-challenge reviewed by **Eduardo**

### The good points are:

* Required gems (i.e. twilio-ruby) are listed in Gemfile
* All tests test real behaviour and not stubs
* The Law of Demeter is respected in the code base
* The Twilio dependency can be injected appropriately
* The design has at least `Takeaway` and `Order` classes or equivalent
* reduce/inject is used to aggregate (e.g. sum) over a collection
* The code follows the Ruby style and coding conventions

### You should consider the following improvements:

* README should be updated
* README should include instructions on how to use application
* Use named subject with `described_class`
* Stubbing the Twilio API
* Unit vs Integration tests
* Explicit tests for every element of the public interface
* Use of modules
* Separation of Concerns
* Personal details and tokens on GitHub
* Open Closed Principle

### Additional comments:
The comments have been posted on my partner's repo. 

see https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md for more details

**Pull request closed without merging**